CHANGES.md

```diff
@@ -4,14 +4,21 @@ Notable changes between releases.
 
 ## Latest
 
-* Build matchbox with Go 1.8 for container images and binaries 
+* New [terraform-provider-matchbox](https://github.com/coreos/terraform-provider-matchbox) plugin for Terraform users!
+* New hosted [documentation](https://coreos.com/matchbox/docs/latest) on coreos.com
+* Add `ProfileDelete`, `GroupDelete`, `IgnitionGet` and `IgnitionDelete` gRPC endpoints
+* Build matchbox with Go 1.8 for container images and binaries
 * Generate code with gRPC v1.2.1 and matching Go protoc-gen-go plugin
+* Update Ignition to v0.14.0 and coreos-cloudinit to v1.13.0
+* Update "fuze" docs to the new name [Container Linux Configs](https://coreos.com/os/docs/latest/configuration.html)
```

Comment:
I figured, matchbox users would mostly be interested in the Container Linux config format itself that they provide to matchbox. Ignition and ct would be internal details they don't worry about too much. With the switch from fuze to Container Linux config in https://github.com/coreos/matchbox/pull/479 I updated those docs so maybe I link to the comparison there via "Learn more"?